pages.es/osx/sntp.md

```diff
@@ -0,0 +1,16 @@
+# sntp
+
+> Un programa cliente dedicado al Simple Network Time Protocol.
```

Comment:
LGTM. Although I would suggest the following:

```suggestion
> Un cliente para el Simple Network Time Protocol.
```

I do not know if the term `programa cliente` is valid.